pkg/ovs/ovsconfig/interfaces.go

```diff
@@ -40,7 +41,7 @@ type OVSBridgeClient interface {
 	CreateAccessPort(name, ifDev string, externalIDs map[string]interface{}, vlanID uint16) (string, Error)
 	CreateInternalPort(name string, ofPortRequest int32, externalIDs map[string]interface{}) (string, Error)
 	CreateTunnelPort(name string, tunnelType TunnelType, ofPortRequest int32) (string, Error)
-	CreateTunnelPortExt(name string, tunnelType TunnelType, ofPortRequest int32, csum bool, localIP string, remoteIP string, psk string, externalIDs map[string]interface{}) (string, Error)
+	CreateTunnelPortExt(name string, tunnelType TunnelType, ofPortRequest int32, csum bool, localIP, remoteIP, dstPort, psk string, erspanOptions, externalIDs map[string]interface{}) (string, Error)
```

Comment:
I'm thinking if it should just use `options` or `extraOptions` to be more generic.

Reply:
Done.